src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -95,6 +95,18 @@ public function getShortDescription(string $class, string $property, array $cont
             }
         }
 
+        foreach ($docBlock->getTagsByName('param') as $param) {
+            if (!$param instanceof DocBlock\Tags\Param) {
+                continue;
+            }
+
+            $paramDescription = $param->getDescription()?->render();
+
+            if (null !== $paramDescription && '' !== $paramDescription) {
```

Comment:
```suggestion
            if ($paramDescription) {
```

Reply:
Are you sure? 

```php
<?php
$paramDescription = '0';

echo $paramDescription ? 'ok' : 'ko'; // 'ko'

echo null !== $paramDescription && '' !== $paramDescription ? 'ok' : 'ko'; // 'ok'
```

Reply:
not sure `0` a useful description either